### DISCUSSION TITLE
Windows 7?

### DISCUSSION THREAD

**Discussion by Sharktnk** [bkaankose/Wino-Mail#442] [General]
Will Wino Mail work on Windows 7? I know, behind the times here ... Can't get the downloaded installer to open. Have tried downloading the zip file but unsure what file to use to install the program. Even tried downloading the v1.8.7 msixbundle but it fails part of the way through (Failed- Needs Authorization}. A bit out of my league here, just wanted to use Wino Mail since WLM has stopped working.

**Comment by bkaankose** [comment]
Unfortunately no. Underlying technology only supports down to Windows 10 for now.
